x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -13,36 +13,57 @@ import {
   SavedObjectsCreateOptions,
   SavedObjectsFindOptions,
   SavedObjectsUpdateOptions,
+  SavedObjectsAddNamespacesOptions,
+  SavedObjectsRemoveNamespacesOptions,
 } from '../../../../../src/core/server';
 import { SecurityAuditLogger } from '../audit';
 import { Actions, CheckSavedObjectsPrivileges } from '../authorization';
+import { CheckPrivilegesResponse } from '../authorization/check_privileges';
+import { SpacesService } from '../plugin';
 
 interface SecureSavedObjectsClientWrapperOptions {
   actions: Actions;
   auditLogger: SecurityAuditLogger;
   baseClient: SavedObjectsClientContract;
   errors: SavedObjectsClientContract['errors'];
   checkSavedObjectsPrivilegesAsCurrentUser: CheckSavedObjectsPrivileges;
+  getSpacesService(): SpacesService | undefined;
```

Comment:
We're relying on the Security plugin's `getSpacesService` method, and this gave me pause:

https://github.com/elastic/kibana/blob/498abb4152c8d22d1bee15b9c357b8daea3261d6/x-pack/plugins/security/server/plugin.ts#L93-L100

https://github.com/elastic/kibana/blob/498abb4152c8d22d1bee15b9c357b8daea3261d6/x-pack/plugins/security/server/plugin.ts#L177-L183

Reply:
`getSpacesService` is meant as a temporary measure until we can figure out how to break the circular dependency between spaces and security. I'd rather not propagate this further, especially if we're not actually using the functionality provided by the spaces service.

Reply:
As discussed offline, we decided to leave this as-is since there's not really a better way of determining whether Spaces is enabled or not during setup.